### PR TITLE
[MIG] account_multicompany_code: Migration to 16.0 T#63809

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
@luisg123v Please review

cc @desdelinux

**Comment by desdelinux** [comment]
@LuisMzz Missing commit history

**Comment by ghost** [comment]
@luisg123v  I already added the commit to `account_multicompany_code`

File: account_multicompany_code/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* account_multicompany_code
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 15.0+e\n"
```
Comment (luisg123v) [inline account_multicompany_code/i18n/es.po]: 16.0.

Please re-generate this file

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
@luisg123v I already made the changes

**Review by luisg123v — APPROVED** [review]
LGTM :+1:

**Pull request merged**